lib/application/contact_form/contact_form_bloc.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:bloc/bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import '../../domain/contact_form/contact_form_contents.dart';
+import '../../domain/contact_form/i_contact_form_api.dart';
+import '../../infrastructure/core/injection.dart';
+
+part 'contact_form_bloc.freezed.dart';
+part 'contact_form_event.dart';
+part 'contact_form_state.dart';
+
+class ContactFormBloc extends Bloc<ContactFormEvent, ContactFormState> {
+  late final IContactFormApi _contactFormApi;
+
+  ContactFormBloc() : super(const _Initial()) {
+    _contactFormApi = getIt<IContactFormApi>();
```

Comment:
If IContactForm is injectable, then just make this as a field, and make the BLoC injectable so that the wrapper for the api is automatically injected.

Reply:
tomorrow ;)